communities/genome/lab/assembly.yml

```diff
@@ -6,12 +6,13 @@ tabs:
     heading_md: >
       Common tools are listed here, or search for more in the full tool panel to the left.
     content:
-      - title_md: <code>Hifiasm</code> - assembly with PacBio HiFi data
+      - title_md: <code>Hifiasm</code> - assembly with PacBio HiFi and/or Nanopore data
```

Comment:
Could you say this more briefly with "long-read data"?

Reply:
I think this is better because I want to emphasise/highlight that this can now use Nanopore data; in the past it was only Hifi